### PR TITLE
Add Azure Function handlers implementation.

### PR DESCRIPTION
**Pull request by sjuarezgx**
Issue:88801

### CODE REVIEW

**Review by jechague — CHANGES_REQUESTED** [review]
Please, do not upload binary files (dotnet/src/dotnetcore/libs/GeneXusServerlessAPI.dll) to the repository.

**Review by claudiamurialdo — APPROVED** [review]

File: .github/workflows/Build.yml [hunk 1]
```diff
@@ -32,6 +32,11 @@ jobs:
       with:
         dotnet-version: '3.1.x'
 
+    - name: Install .NET Core 5.0
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '5.0.101'
+
```
Comment (jechague) [inline .github/workflows/Build.yml:39]: Can we stop installing .Net Core 3.1? And what about using a float number for the patch version?

File: dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   </configSections>
+   <datastores>
+      <add  key="Connection-Default-DBMS" value="sqlserver"/>
+      <add  key="Connection-Default-Port" value=""/>
+      <add  key="Connection-Default-LockTimeout" value="0"/>
+      <add  key="Connection-Default-LockRetryCount" value="10"/>
+      <add  key="Connection-Default-IsolationLevel" value="CR"/>
+      <add  key="Connection-Default-Datasource" value="LcSFozUYYEwCMNTWIBzPxzBLm+G9NbFezqdOu8D8G4z="/>
+      <add  key="Connection-Default-User" value="L3KpX01Y+7yriRShRiH2vS=="/>
+      <add  key="Connection-Default-Password" value="L3KpX01Y+7yriRShRiH2vS=="/>
```
Comment (jechague) [inline dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config:19]: Are these valid when used outside the tests?

File: dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      <add  key="DateFormat" value="MDY"/>
+      <add  key="YearLimit" value="1940"/>
+      <add  key="TimeAmPmFormat" value="12"/>
+      <add  key="ENABLE_MANAGEMENT" value="0"/>
+      <add  key="MAX_CURSOR" value="100"/>
+      <add  key="STATIC_CONTENT" value=""/>
+      <add  key="GX_BUILD_NUMBER" value="8504161"/>
+      <add  key="CACHE_CONTENT_EXPIRATION" value="36"/>
+      <add  key="FTP_DEFAULT_USER" value="Zd8hx1OPJaolo5nN5s4nmc=="/>
+      <add  key="FTP_DEFAULT_PASSWORD" value="kvwmxbHzjMA66dQNAHRlhn=="/>
```
Comment (jechague) [inline dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config:93]: What about these values?

**Comment by cinthamo** [comment]
<details>
<summary>Cherry pick to beta failed, 1 conflicted file</summary>
<ul>
	<li>.github/workflows/Build.yml</li>
</ul>
</details>

File: .github/workflows/Build.yml (hunk 1 above)
Comment (claudiamurialdo) [reply .github/workflows/Build.yml]: Absolutely. Done

File: dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config (hunk 2 above)
Comment (claudiamurialdo) [reply dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config:19]: They aren't. Removed encrypted values.

File: dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config (hunk 3 above)
Comment (claudiamurialdo) [reply dotnet/test/DotNetExtensions/AzureFunctions/test/client.exe.config:93]: They aren't. Removed encrypted values.

**Comment by cinthamo** [comment]
<details>
<summary>Manual cherry pick to beta success</summary>
<ul>
	<li>75a9f85</li>
	<li>8fb069a</li>
</ul>
</details>

**Review by claudiamurialdo — APPROVED** [review]

**Review by claudiamurialdo — APPROVED** [review]

**Review by jechague — APPROVED** [review]
The DLL is no longer here and the changes to the build files are OK.

LGTM

**Pull request merged**